encfs/main.cpp

```diff
@@ -190,6 +191,7 @@ static bool processArgs(int argc, char *argv[],
   out->isVerbose = false;
   out->idleTimeout = 0;
   out->fuseArgc = 0;
+  out->syslogTag.assign("encfs");
```

Comment:
Because option `p)` some lines below uses `assign` :)
I updated with `=` which is more convenient 👍